assets/src/application/ui/layout/dropdownMenu/DropdownMenu.tsx

```diff
@@ -13,11 +13,10 @@ const DropdownMenu: React.FC<DropdownMenuProps> = ({ children, className, menuLi
 			{({ isOpen, onClose }) => (
 				<div className='ee-dropdown-menu'>
 					<DropdownToggle isOpen={isOpen} onClose={onClose} {...toggleProps} />
-					<div className='ee-dropdown-menu__list-wrapper'>
```

Comment:
this `div` was the problem, it seems that chakra is a bit strict about what elements needs to be nested